production/helm/loki/templates/gateway/configmap-gateway.yaml

```diff
@@ -1,4 +1,4 @@
-{{- $isSimpleScalable := eq (include "loki.deployment.isSimpleScalable" .) "true" -}}
+{{- $isSimpleScalable := eq (include "loki.deployment.isScalable" .) "true" -}}
```

Comment:
using this opportunity to get away from the terms "Simple Scalable". This has confused community members who figured "Simple" meant not production ready.